package/firmware/linux-firmware/intel.mk

```diff
@@ -202,7 +202,7 @@ $(eval $(call BuildPackage,iwlwifi-firmware-ax210))
 Package/iwlwifi-firmware-be200 = $(call Package/firmware-default,Intel BE200 firmware)
```

Comment:
While you are at it please update the `iwlwifi-ty-a0-gf-a0-83.ucode ` file to version 89 too.